loc/ui/inventory.js

```diff
@@ -9,7 +9,14 @@ export default {
     fr: 'inventaire',
     nl: 'inventarisatie',
     pt: 'inventário',
-    it: 'merci in magazzino',
-    pl: 'inwentaryzacja'
+    it: 'inventario',
+    pl: 'Inwentarz',
+    bg: 'складова наличност',
+    cs: 'inventář',
+    el: 'καταγραφή εμπορευμάτων',
+    ja: '在庫',
+    ph: 'imbentaryo',
+    se: 'lager',
```

Comment:
```suggestion
    se: 'lager'
```

You also left an empty line below. Perhaps you missed a translation.